about.css

```diff
@@ -0,0 +1,334 @@
+  bottom: -10px;
+}
+
+.hamburger.active span {
+  transform: rotate(45deg);
+}
+
+.hamburger.active span::before {
+  top: 0;
+  transform: rotate(90deg);
+}
+
+.hamburger.active span::after {
+  bottom: 0;
+  transform: rotate(90deg);
+}
+
+.Logo-sec {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  gap: 20px;
+  padding: 20px;
+  border-radius: 20px;
+}
+
+.Logo-sec hr {
+  width: 100%;
+  color: #9a9898;
+}
+
+footer img {
+  width: 100px;
+}
+
+.Logo-sec img {
+  width: 200px;
+  height: 200px;
+}
+
+.Logo-sec h2 {
+  color: #9a9898;
+}
+
+.about-inviro {
+  text-align: center;
+}
+
+.albums {
+  text-align: center;
+  padding: 24px;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  background-color: #3b3e45;
+}
+
+.albums h2 {
+  color: whitesmoke;
+}
+
+.albums img {
+  width: 100%;
+}
+
+footer {
+  display: flex;
+  align-items: center;
+  border: #b3b3b3 solid 2px;
+  justify-content: space-evenly;
+}
+
+footer h6 {
+  font-size: 16px;
+  width: 50%;
+}
+
+/* Dropdown menu styles */
+.dropdown-menu {
+  display: none;
+  background-color: #f8f8f8;
+  list-style-type: none;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 999;
+  padding-top: 200px;
+  background-image: url(Images/background-op1.png);
+  background-size: cover;
+}
+
+.dropdown-menu a {
+  font-size: 30px;
+  font-weight: 700;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  color: #ff5438;
+  padding: 10px;
+  border-bottom: none;
+  transition: background-color 0.3s ease-in-out;
+}
+
+.dropdown-menu.active {
+  display: block;
+}
+
+.dropdown-menu li {
+  padding: 10px;
+  transition: background-color 0.3s ease-in-out;
+}
+
+.dropdown-menu li:hover {
+  background-color: #ddd;
+}
+
+@media (min-width: 769px) {
+  .hamburger,
+  .about-para {
+    display: none;
+  }
+
+  .nav-box {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .grey-bar ul {
+    background-color: #343434;
+    display: flex;
+    justify-content: flex-end;
+    padding: 10px 8%;
+    list-style: none;
+    gap: 20px;
+    color: #fff;
+  }
+
+  .the-bar {
+    padding: 0 8%;
+    display: flex;
+    align-items: center;
+    gap: 30%;
+  }
+
+  .the-bar img {
+    width: 100px;
+  }
+
+  .dropdown-desktp {
+    display: flex;
+    gap: 80px;
+  }
+
+  .dropdown-desktp a {
+    text-decoration: none;
+    color: black;
+  }
+
+  .about-head {
+    height: 723px;
+    padding-top: 115px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .about-head h3 {
+    font-size: 60px;
+    font-weight: 900;
+    color: #ff5438;
+  }
+
+  .about-head h2 {
+    font-size: 60px;
+  }
+
+  .about-paDsk {
+    width: 793px;
+    height: 175px;
+    font-size: 25px;
+    font-weight: 400;
+    background-color: white;
+    margin-top: 30px;
+  }
```

Comment:
- [ ] For a good user experience kindly reduce the `font-size` of the paragraph text`(highlighted below)` so it looks like one of the provided designs.

<table>
  <tr>
    <td align="middle">Your design</td>
     <td align="middle">Provided design</td>
  </tr>
  <tr>
    <td valign="top">
<img width="1440" alt="Screenshot 2023-04-24 at 22 27 17" src="https://user-images.githubusercontent.com/107192076/234130621-8dad29fa-f5a6-4964-b5a9-8e229d3650e4.png">
     </td>
    <td align="top">
<img width="710" alt="Screenshot 2023-04-24 at 22 28 34" src="https://user-images.githubusercontent.com/107192076/234130641-c706c4ea-d4e4-4b30-b630-c3f040ab8dbb.png">
    </td>
  </tr>
 </table>